certbot/certbot/_internal/tests/helpful_test.py

```diff
@@ -117,92 +113,5 @@ def test_add_group_topic_requested_help(self):
             assert arg_parser.groups["certonly"] is False
 
 
-class TestParseArgsErrors:
```

Comment:
Why is this test removed?

Reply:
i mentioned in a commit message, but:

    We were already testing most of these cases in cli_test.py, only
    with a more complete HelpfulArgumentParser setup. And since the hsts/no-hsts
    test was manually performing the kind of argument adding that cli
    already does out of the box, I figured the cli tests were a more natural
    place for it.

Reply:
ah, didn't realize there was info I was missing in the commit messages! I see it now, looks good.